eng/Versions.props

```diff
@@ -49,7 +49,7 @@
     <!--
       TODO: Remove pinned version once arcade supplies a compiler that enables the repo to compile.
     -->
-    <MicrosoftNetCompilersToolsetVersion>4.4.0-1.22328.22</MicrosoftNetCompilersToolsetVersion>
+    <MicrosoftNetCompilersToolsetVersion>4.4.0-1.22356.12</MicrosoftNetCompilersToolsetVersion>
```

Comment:
I noticed this dependency isn't listed in Version.Details.xml, do we ever plan on having it be automatically updated by arcade's dependency flow? I assume the answer is no, but just wanted to ask.

Reply:
Another question, does this change affect at all the minimum version required in order to use .NET 7 preview 7? In other words, if I'm using .NET 7 preview 6 in the oldest version of VS that is required by it, will I get broken when ingesting preview 7?

Reply:
This doesn't affect VS minimums (unlike the Microsoft.CodeAnalysis.* ones that do).